content/backmatter.tex

```diff
@@ -234,7 +234,7 @@ \section{\ac{MPI} Interoperability}
 \openshmem routines can be used to communicate with processes running from the
 same or different executable files, provided that the communication is limited
 to symmetric data objects.  On these systems, static memory such as a
-\Fortran{} common block or \Clang{} global variable, is symmetric between
+\Fortran{} common block or \Cstd global variable, is symmetric between
```

Comment:
Yes, I think it would be good if we're consistent across the board. I'll make a note to address that.